package.json

```diff
@@ -36,7 +38,13 @@
     "url": "git+https://github.com/mozilla/observatory-cli.git"
   },
   "scripts": {
+    "lint": "npm-run-all lint:*",
```

Comment:
Calling `$ npm run lint` (auto called by `npm test` as a `pretest` script), will now run all the `lint:*` prefixed scripts. :magic: